<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_username">
     <property name="text">
      <string>Username:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_username</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_username"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_password">
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_password</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_new">
       <property name="text">
        <string>New User</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_reset">
     <property name="text">
      <string>Forgot Password?</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_username</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>LoginDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_username</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit_password</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_username</sender>
   <signal>returnPressed()</signal>
   <receiver>lineEdit_password</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
